js/tic_tac_toe.js

```diff
@@ -1,2 +1,84 @@
-// Your code here
-// Use the jQuery
+var count = 0;
+var blue_rows = [];
+var red_rows = [];
+var blue_columns = [];
+var red_columns = [];
+var $lFirst, $lSecond, $lThird, $rFirst, $rSecond, $rThird, $heading;
+
+$(document).ready(function() {
+  $cell = $('.box_cell');
+  $lFirst = $('#box_0_0');
+  $lSecond = $('#box_1_1');
+  $lThird = $('#box_2_2');
+  $rFirst = $('#box_2_0');
+  $rSecond = $('#box_1_1');
+  $rThird = $('#box_0_2');
+  $heading = $('h1');
+  $cell.on('click', setColor);
+});
+
+var changeBlue = function() {
```

Comment:
changeBlue and changeRed work well. Any thoughts of you could make them into one function that isn't too complex inside?